### ISSUE TITLE
Losen org.springframework.validation.Errors from binding/target object [SPR-14013]

### ISSUE THREAD

**Issue by spring-issuemaster** [spring-projects/spring-framework#18585]
**[Marcel Overdijk](https://jira.spring.io/secure/ViewProfile.jspa?name=marceloverdijk)** opened **[SPR-14013](https://jira.spring.io/browse/SPR-14013?redirect=false)** and commented

I think this is gonna be a longshot, but nevertheless I want to raise it.

I really like the `org.springframework.validation.Errors` interface since the early days developing jsp webapps. In that time it would make absolute sense to have a target object bound to the `Errors` instance. As to render values again in jsp forms.

But now with REST architecture I still like the `Errors` interface but the fact that it is tight to a target object is not always desired.
I could of course create a custom implementation of the `Errors` interface but it contains methods like `getObjectName`, `getFieldType`, `getFieldValue` which in this case are not desired.

Basically what I'm asking is to have an interface which is completely loosened from target object.
It would also be nice to have simple (abstract) class which stores errors etc, but without having to bind a target object. All existing implementations of `Errors` require binding a target object.

---

**Affects:** 4.2.5

**Comment by spring-issuemaster** [comment]
Bulk closing outdated, unresolved issues. Please, reopen if still relevant.
